ledger/metrics/src/test/suite/scala/com/daml/metrics/AkkaInstrumentedSpec.scala

```diff
@@ -125,7 +119,7 @@ final class InstrumentedSourceSpec extends AsyncFlatSpec with Matchers with Akka
   def samplePercentage(samples: List[Long])(filter: Long => Boolean): Double =
     samples.count(filter).toDouble / samples.size.toDouble * 100.0
 
-  it should "signal mostly full buffer if slow consumer" ignore {
```

Comment:
are you sure we can enable these? not much changed with the implementation, and that was flaky in tests :thinking:

Reply:
I left two disabled. But on a second thought I would delete them altogether. The others I would like to give one more chance (the ones that are asserting full or empty buffer)